master/internal/db/postgres_jobs.go

```diff
@@ -0,0 +1,19 @@
+package db
+
+import (
+	"github.com/jmoiron/sqlx"
+	"github.com/pkg/errors"
+
+	"github.com/determined-ai/determined/master/pkg/model"
+)
+
+// addJob persists the existence of a task from a tx.
+func addJob(tx *sqlx.Tx, t *model.Job) error {
```

Comment:
I think we have to delete jobs from this table that are completed or deleted. Will this be added later?

Reply:
we had a [discussion](https://determined-community.slack.com/archives/C0269E3SMJP/p1630365373008500?thread_ts=1630362854.004500&cid=C0269E3SMJP) about this in #queue-management the plan is to leave and never delete for future use.

Reply:
ah I didn't pay attention to that. I'll need to think about this a bit more - at first I thought it wasn't a good idea, but I'm now thinking it's not that bad. Esp. if we only refer to active jobs in the first place.

Reply:
conclusion: probably not very bad like I initially thought

Reply:
let's continue in the channel if needed